iml-wasm-components/iml-stratagem/src/lib.rs

```diff
@@ -68,6 +69,7 @@ impl Default for Model {
         Model {
```

Comment:
Just curious, why do we need a default implementation for this?

Reply:
I was actually thinking about that as well. Maybe I should remove this altogether.